src/components/TaxonomyTerms/TaxonomyTerms.module.scss

```diff
@@ -0,0 +1,25 @@
+@import "styles/Variables";
+
+.taxonomy {
+  font-weight: 700;
```

Comment:
Instead of hardcoding 700 here, you should use a variable $font-weight-extra-bold